docs/modules/user-guide/partials/proc_adding-attributes-to-a-devfile.adoc

```diff
@@ -15,7 +15,7 @@ Devfile attributes can be used to configure various features.
 
 . Define a custom attribute
 +
-When an editor is not specified in a devfile, a default is provided. The default value of `false` means that the devfile requests the provisioning of the default editor.
+When an editor is not specified in a devfile, a default is provided. Also, when an editor is not specified, consider a user defined use case. When no editor is needed, use the `editorFree` attribute.
```

Comment:
Dont think this makes much sense to me, I just wanted to convey something along the lines of:

Consider a use case when no editor is provided. When no editor is specified, a default editor is provided. To represent this user defined example, use the `editorFree` attribute as shown below.